src/components/CompactInformationPanel/CompactInformationPanel.tsx

```diff
@@ -68,11 +68,13 @@ const Wrapper = styled(Base)<{ themes: Theme }>`
     themes: {
       fontSize: { pxToRem },
       spacing,
+      shadow,
     },
   }) => {
     return css`
       display: flex;
       padding: ${pxToRem(spacing.XS)};
+      box-shadow: ${shadow.DIALOG};
```

Comment:
imo: 個人的なこだわりで影響が外に向いているプロパティを上の方に書くようにしています

```suggestion
      display: flex;
      box-shadow: ${shadow.DIALOG};
      padding: ${pxToRem(spacing.XS)};
```